app/build.gradle

```diff
@@ -37,13 +42,15 @@ android {
             buildConfigField "boolean", "isInternetAvailable", "false"
             resValue "string", "app_name", "GoIV Offline"
             ext.enableCrashlytics = false
+            versionName "2.2.0 (Offline)"
         }
 
         online {
             applicationId "com.kamron.pogoiv"
             buildConfigField "boolean", "isInternetAvailable", "true"
             resValue "string", "app_name", "GoIV"
             ext.enableCrashlytics = true
+            versionName "2.2.0"
```

Comment:
Is the versionName actually used? Still, I'd prefer to use the filename, which is more free. The current one, `GoIV-Online.2.2.0.apk` isn't `${app_name}.${version}.apk`.